core/store/orm/orm.go

```diff
@@ -665,20 +665,26 @@ func (orm *ORM) IdempotentInsertEthTaskRunTx(taskRunID models.ID, fromAddress co
 	}
 }
 
-// EthTransactions returns all transactions limited by passed parameters.
-func (orm *ORM) EthTransactionsWithOrderedAttempts(offset, limit int) ([]models.EthTx, int, error) {
-	count, err := orm.CountOf(&models.EthTx{})
+// EthTransactionsWithAttempts returns all eth transactions with at least one attempt
+// limited by passed parameters. Attempts are sorted by created_at.
+func (orm *ORM) EthTransactionsWithAttempts(offset, limit int) ([]models.EthTx, int, error) {
+	ethTXIDs := orm.DB.Select("DISTINCT eth_tx_id").Table("eth_tx_attempts").QueryExpr()
+
+	var count int
+	err := orm.DB.Table("eth_txes").Where("id IN (?)", ethTXIDs).Count(&count).Error
```

Comment:
Cool! Didn't know you could do this.

An alternative I've been using is `Where("id = ANY(?)", pq.Array(ethTXIDs))`